sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -2950,6 +2982,117 @@ private void SetNameFieldsIfNull()
             }
         }
         #endregion SetAccessTier
+
+        private async Task<Stream> ClientSideDecryptInternal(
+            Stream content,
+            Metadata metadata,
+            HttpRange originalRange,
+            string receivedContentRange,
+            bool async,
+            CancellationToken cancellationToken)
+        {
+            ContentRange? contentRange = string.IsNullOrWhiteSpace(receivedContentRange)
+                ? default
+                : ContentRange.Parse(receivedContentRange);
+
+            EncryptionData encryptionData = GetAndValidateEncryptionDataOrDefault(metadata);
+            if (encryptionData == default)
+            {
+                return content; // TODO readjust range
```

Comment:
Is this still relevant?

Reply:
Yes. Yes it was. Addressed in upcoming push.